### PR TITLE
Love

### PR DESCRIPTION
**Pull request by PaeP3nguin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,7 @@
 - [fuzzco](http://fuzzco.com)
 
 ## people
+- [paep3nguin](https://github.com/PaeP3nguin)
```
Comment (thehandsomepanther) [inline README.md:29]: this is NOT a cool site. please amend

File: CONTRIBUTION.md [hunk 2]
```diff
@@ -1,2 +1,2 @@
 # contributing
-all pull requests will be reviewed by josh shi except for those from WILL XIAO
+all pull requests will be reviewed by josh shi and approved because I love you
```
Comment (thehandsomepanther) [inline CONTRIBUTION.md:2]: has this passed unit tests?

**Review by thehandsomepanther — CHANGES_REQUESTED** [review]

File: CONTRIBUTION.md (hunk 2 above)
Comment (PaeP3nguin) [reply CONTRIBUTION.md:2]: Yeah, it has a 1600 SAT score

File: README.md (hunk 1 above)
Comment (PaeP3nguin) [reply README.md:29]: I'll fix in the next PR

**Comment by thehandsomepanther** [comment]
closed because this violates our community guidelines

**Comment by PaeP3nguin** [comment]
This is very unfriendly

**Pull request closed without merging**